src/main/webapp/search-style.css

```diff
@@ -0,0 +1,42 @@
+.hidden {
+  display: none;
+}
+
+label, select {
+  margin-bottom: 20px;
+}
+
+#login-box {
+    margin:auto;
+    width: 120px;
+    padding: 30px;
+}
+
+#search-textarea, #search-selectboxes, #class-textarea, #class-selectboxes {
+    display: flex;
+}
+
+textarea, select, fieldset {
```

Comment:
Keep in mind that this will affect everything on the page (and thus can easily conflict). Often we prefer to put styles like this in the top level index.css file or just don't use them at all.

I'll let it slide for now.

Reply:
Since I'm only calling this in search.html, I figured it would only affect those things on search.html (which is what I intended) - or is that still bad style?

Reply:
It's still risky if you end up adding other css files to search.html but much better than I was thinking. I'm too used to single page apps haha. Don't worry about changing it.